day03/ex00/ex00.md

```diff
@@ -0,0 +1,143 @@
+
+- **variable expense** : we don't need to invest in huge data centers you may not use. You pay for how much you consume !
+- **available in minutes** : new IT resources can be accessed within minutes.
+- **economies of scale** : because of the number of users. Cloud providers can achieve higher economies of scale translating in lower prices.
+- **global in minutes** : cloud architectures can be deployed really easily all around the world.
+
+Deployments using the cloud can be `all-in-cloud-based` (the entire infrastructure is in the cloud) or `hybrid` (using on premise and cloud).
+
+## AWS global infrastructure
+
+Amazon Web Service (AWS) is a cloud service provider, also known as infrastructure as a service (`IaaS`). AWS is the clear market leader in this domain and offers many services compared to other cloud providers.
+
+AWS has some interesting proporties as :
+
+- **High availability** : Any file can be accessed from anywhere
+- **Fault tolerance** : If an AWS server fails. You can still retrieve the file (the fault tolerancy is due to redundancy)
+- **Scalibility** : Possibility to add more servers when needed
+- **Elasticity** : Possibility to grow or shrink infrastructure
+
+AWS provides a highly available technology infrastructure platform with multiple locations worldwide. These locations are composed of `regions` and `availability zones`.
+
+Each region is a separate geographic area. Each region has multiple, isolated locations known as availability zones. An availibity zone is a physical data center geographically seperated from other availibility zones (redundant power, networking and connectivity).
+
+We can achieve high availability by deploying your application across multiple availability zones.
+
+![AWS regions](../assets/aws_regions.png){width=400px}
+
+The `edge locations` you see on the picture are endpoints for AWS which are used for caching content (performance optimization mechanism in which data is delivered from the closest servers for optimal application performance). Typically consists of CloudFront (Amazon's content delivery network (CDN)).
+
+## AWS account registration
+
+When we create a new account on AWS, we have an AWS Free Tier. It allows us to use some of AWS resources for free each month during one year.
+
+In order to create an AWS account, we must go to the following [address](https://aws.amazon.com).
+
+Then we click on the create an account button and fill the following form.
+
+![Account creation 1](../assets/account_creation_1.png){width=240px}
+
+\newpage
+
+![Account creation 2](../assets/account_creation_2.png){width=245px}
+
+Don't worry! Even if we enter our card number, this day is not going to cost us anything. First,  Indeed, AWS as a free tier usage which allows us to use a small amount of AWS resources for free. This will be sufficient for what we want to do during this day. At the end of your day you will need to destroy your infrastructure entirely (don't keep things runing) !!!
```

Comment:
```suggestion
Don't worry! Even if you enter your card number, this day is not going to cost you anything. First, indeed, AWS as a free tier usage that allows you to use a small amount of AWS resources for free. This will be sufficient enough for what you are going to do today. By the end of the day, you will have to entirely destroy your infrastructure (don't keep things running) !!!
```